content/en/synthetics/private_locations/configuration.md

```diff
@@ -232,7 +232,7 @@ Format log output between `"compact"`, `"pretty"`, `"pretty-compact"`, and `"jso
 `--verbosity`
 : **Type**: Number <br>
 **Default**: `3`<br>
-Verbosity level (for example, `-v`, `-vv`, and `-vvv`).
+Verbosity level from `1` (errors only) to `4` (debug logs and above). Setting the verbosity from the command line is done with `-v`, `-vv`, `-vvv`, and `-vvvv` arguments.
```

Comment:
**Question for reviewers:** should we include the table provided in the PR description to make this option easier to understand? It is quite hard to convey the proper usage between CLI arguments and the configuration file 🤔